docs/release-notes/v0.27.0.md

```diff
@@ -0,0 +1,126 @@
+* Publish edge rad CLI binaries to `./rad` instead of `./dist/os_arch/release/rad` by @willdavsmith in https://github.com/radius-project/radius/pull/6634
+* Add extra logging and retries to Test_Gateway by @willdavsmith in https://github.com/radius-project/radius/pull/6622
+* Fix windows edge rad CLI publishing by @willdavsmith in https://github.com/radius-project/radius/pull/6646
+* Show recipe template kind and path in the status by @vinayada1 in https://github.com/radius-project/radius/pull/6450
+* Update release verification workflow to use GH releases by @willdavsmith in https://github.com/radius-project/radius/pull/6616
+* Enable OpenSSF Scorecard by @kachawla in https://github.com/radius-project/radius/pull/6637
+* Fix string formatting in pod status check assertion by @kachawla in https://github.com/radius-project/radius/pull/6656
+* Add steps for Radius Bicep release by @willdavsmith in https://github.com/radius-project/radius/pull/6606
+* Fix flaky TF recipe tests by @sk593 in https://github.com/radius-project/radius/pull/6644
+* Switch ACR Helm chart to GHCR for rad init by @willdavsmith in https://github.com/radius-project/radius/pull/6510
+* Hard-coding dev recipes, refactoring its logic, and adding a functional test by @ytimocin in https://github.com/radius-project/radius/pull/6513
+* Release 0.27: Create RC release by @kachawla in https://github.com/radius-project/radius/pull/6662
+* Revert "Switch ACR Helm chart to GHCR for rad init (#6510)" by @willdavsmith in https://github.com/radius-project/radius/pull/6668
+* Update Helm Chart Download Error Message by @kachawla in https://github.com/radius-project/radius/pull/6671
+* Use GHCR for Radius Helm repository by @willdavsmith in https://github.com/radius-project/radius/pull/6672
+* Change arch of release verification script to amd64 by @willdavsmith in https://github.com/radius-project/radius/pull/6665
+* Release 0.27: Create RC2 release by @kachawla in https://github.com/radius-project/radius/pull/6677
+* Fix redirection in bash script by @youngbupark in https://github.com/radius-project/radius/pull/6676
+* Release 0.27: Create RC3 release by @kachawla in https://github.com/radius-project/radius/pull/6678
+* Release 0.27: Create RC4 release by @kachawla in https://github.com/radius-project/radius/pull/6679
+
+## New Contributors
+* @wmeints made their first contribution in https://github.com/radius-project/radius/pull/6529
+* @daixiang0 made their first contribution in https://github.com/radius-project/radius/pull/6526
+* @thapasusheel made their first contribution in https://github.com/radius-project/radius/pull/6543
+* @shubham1172 made their first contribution in https://github.com/radius-project/radius/pull/6550
+* @Mayureshd-18 made their first contribution in https://github.com/radius-project/radius/pull/6596
+
+**Full Changelog**: https://github.com/radius-project/radius/compare/v0.26.9...v0.27.0
```

Comment:
Should the changelog be from v0.26.0 to v0.27.0 instead? I think 0.26.9 to 0.27.0 wouldn't have all of the changes for all the 0.26 versions

Reply:
I'd assume we would want to include changes since the most recent release even if it was a patch release. But I don't have a strong preference here, @AaronCrawfis do you have any suggestions on this?

Reply:
Yep the comparison should be between this version and the latest minor release, instead of the latest patch release.